packages/code-studio/src/plugins/PluginUtils.tsx

```diff
@@ -58,21 +58,21 @@ class PluginUtils {
   static async loadJson(
     jsonUrl: string
   ): Promise<{ plugins: { name: string; main: string }[] }> {
-    return new Promise((resolve, reject) => {
-      const request = new XMLHttpRequest();
-      request.addEventListener('load', () => {
-        try {
-          const json = JSON.parse(request.responseText);
-          resolve(json);
-        } catch (err) {
-          reject(err);
+    return fetch(jsonUrl).then(res => {
```

Comment:
Write this with await to be consistent, rather than mixing .then chaining with async/await.
```
const res = await fetch(jsonUrl);
...
```